content/en/platform/corda/5.0-beta/corda-cli-reference/rest-permission-templates.md

```diff
@@ -2,14 +2,14 @@
 date: '2023-01-10'
 menu:
   corda-5-beta:
-    identifier: corda-5-beta-rpc-permissions-templates
+    identifier: corda-5-beta-rest-permissions-templates
     weight: 6000
     parent: corda-5-cli-reference
 section_menu: corda-5-beta
-title: "RPC Permission Templates"
+title: "REST Permission Templates"
```

Comment:
I think these should be called RBAC Permission Templates, and same for change in the overview. This is because we use a role based access control (RBAC) permissioning system to authorize and authenticate users of our REST endpoints

Reply:
> I think these should be called RBAC Permission Templates, and same for change in the overview. This is because we use a role based access control (RBAC) permissioning system to authorize and authenticate users of our REST endpoints

I wasn't sure about this change so thanks for clarifying.